src/OnboardingSPA/data/flows/default.js

```diff
@@ -35,6 +36,7 @@ export const getSteps = ( chapters = initialChapters ) => {
 export const getRoutes = ( chapters = initialChapters ) => {
 	let routes = [ ...pages ];
 	routes.push( stepTheFork );
+	routes.push( stepMigration );
```

Comment:
We should inject this step only once the button is clicked, not by default. This way we will respect the capability check as well. Same with `src/OnboardingSPA/data/flows/ecommerce.js` and `src/OnboardingSPA/data/flows/sitegen.js`.